src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc

```diff
@@ -1,27 +1,21 @@
 [id="configuring-namespace-strategies_{context}"]
 = Configuring namespace strategies
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+Word 'namespace' is used interchangeably with OpenShift's 'project'.
 
-[discrete]
-== Prerequisites
+=== Namespace per Workspace strategy
+TODO
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
+=== All workspaces in one namespace strategy
+TODO
 
-[discrete]
-== Procedure
+=== One Namespace per User strategy
+This strategy provides more security than ‘One namespace for all workspaces strategy’, by isolating user to his/her own namespace. At the same time it’s not so resource heavy as ‘Namespace per workspace strategy’.
```

Comment:
source of this is in original issue description https://github.com/eclipse/che/issues/13488#issue-453044497
 > Use one namespace for all the workspace of every Che user (security issue)
I'm reasoning that it has better security because of isolating user. However, I can't reason how it is more flexible.
I would like to see comment from doc team, whether they feel the same that "more security" can be misinterpreted. @rkratky

Reply:
'more security' is indeed ambiguous.